### PR TITLE
language and markup tweaks

### PR DESCRIPTION
**Pull request by FND**
Just a few pedantic details; feel free to reject changes at will. (It might be easier to review commits individually - perhaps even locally with `git diff --ignore-all-space --color-words`.)

PS: This is an excellent article, I really enjoyed your analysis - thanks for sharing it.

### CODE REVIEW

**Comment by gustafnk** [comment]
Super nice! 👍 I'll have a look

File: index.md [hunk 1]
```diff
@@ -253,7 +253,7 @@ Using root relative URLs (i.e.' /path/to/resource') to external resources is a b
 [Edge-Side Include](https://en.wikipedia.org/wiki/Edge_Side_Includes) (ESI) is a technology that provides a declarative way to include content on the server-side, like this:
 
 ```html
-<esi:include src="/shopping-cart" />
+<esi:include src="/shopping-cart">
```
Comment (gustafnk) [inline index.md]: ESI tags need to be self-closing:

> Remember the trailing slash in <esi:include src="something.html"/>, it must be a self-closing XML element.

 (from https://www.varnish-cache.org/trac/wiki/ESIfeatures)

Comment (gustafnk) [inline index.md]: I just noticed that I have other places where I don't close the ESI element. I'll fix that...

Comment (FND) [inline index.md]: Ah, I didn't know that - good catch.

**Pull request merged**